app/models/user.rb

```diff
@@ -3,7 +3,7 @@ class User < ApplicationRecord
 
   validates :email, uniqueness: true, presence: true
   validates :name, presence: true
-  validates :password, presence: true, require: true
+  validates :password, presence: true
```

Comment:
The has_secure_password is doing the 'require: true' under the hood. Good to know.